core/lib/config/src/configs/contracts.rs

```diff
@@ -49,6 +51,7 @@ pub struct ContractsConfig {
     pub fri_recursion_leaf_level_vk_hash: H256,
     pub prover_at_genesis: ProverAtGenesis,
     pub snark_wrapper_vk_hash: H256,
+    pub base_token_addr: Address,
```

Comment:
This is a mandatory config parameter, which means that it _has_ to be set before the server launches.
If we'll roll this change out without updating configs beforehand, the servers will enter the crashloop.

Would it make sense to use `#[serde(default = "...")]` here to implement a graceful fallback to ETH token if the variable is not set?

Reply:
This is now an Option<Address> so this is not needed.